src/js/components/Drop/DropContainer.js

```diff
@@ -123,30 +123,24 @@ export class DropContainer extends Component {
     const { align, dropTarget, responsive, stretch, theme } = this.props;
     const windowWidth = window.innerWidth;
     const windowHeight = window.innerHeight;
-
     /* eslint-disable-next-line react/no-find-dom-node */
     const target = findDOMNode(dropTarget);
     /* eslint-disable-next-line react/no-find-dom-node */
     const container = findDOMNode(this.dropRef.current);
     if (container && target) {
       // clear prior styling
-      container.style.left = '';
       container.style.width = '';
-      container.style.top = '';
-      container.style.maxHeight = '';
-
+      container.style.height = '';
```

Comment:
We don't set `height`, why do we need this?

Reply:
Yeah i think this was a mistake on my side. It should be maxHeight

Reply:
fixed